<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Parametros:</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tipo</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>% Mutacion Ind</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>% Mutacion Gen</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Poblacion Inicial</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Poblacion Maxima</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Espacio</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Costo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createPackage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Crear Paquete</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Precio Publico</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="typePackage">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="spacePackage">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="costPackage">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="pricePackage">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="mutInd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="mutGen">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="poInitial">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="poMax">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="findMax">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Maximizar</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Generaciones</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="generations">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="title_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tama√±o contenedor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tamContenedor">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
